src/main/flight/flight.c

```diff
@@ -96,14 +96,38 @@ static void pidBaseflight(pidProfile_t *pidProfile, controlRateConfig_t *control
 {
     float RateError, errorAngle, AngleRate, gyroRate;
     float ITerm,PTerm,DTerm;
+    int32_t stickPosAil, stickPosEle, mostDeflectedPos;
     static float lastGyroRate[3];
     static float delta1[3], delta2[3];
     float delta, deltaSum;
     float dT;
     int axis;
+    float horizonLevelStrength = 1;
 
     dT = (float)cycleTime * 0.000001f;
 
+    if (FLIGHT_MODE(HORIZON_MODE)) {
+
+        // Figure out the raw stick positions
+        stickPosAil = getRcStickPosition(FD_ROLL);
+        stickPosEle = getRcStickPosition(FD_PITCH);
+
+        if(abs(stickPosAil) > abs(stickPosEle)){
+            mostDeflectedPos = abs(stickPosAil);
+        }
+        else {
+            mostDeflectedPos = abs(stickPosEle);
+        }
```

Comment:
Using single max() would be equally readable ;-)

`mostDeflectedPos = max(abs(getRcStickPosition(FD_ROLL)), abs(getRcStickPosition(FD_PITCH)));`